prac_05/word_occurences.py

```diff
@@ -0,0 +1,20 @@
+column_width = 0
+
+text = input("Enter in text: ")
+
+words = text.strip().split()
+
+count_words = {}
+
+for word in words:
+    if word in count_words:
+        count_words[word] += 1
+    else:
+        count_words[word] = 1
+
+for word in words:
+    if len(word) > column_width:
+        column_width = len(word)
```

Comment:
You might also be able to merge this for loop with the one above, because they are both iterating through the words list